### ISSUE TITLE
support for valueRecorder?

### ISSUE THREAD

**Issue by Nan2018** [GoogleCloudPlatform/opentelemetry-operations-python#67]
it seems currently `valueRecoder` is not supported. Any plan for it?

**Comment by aabmass** [comment]
Yes hoping to support it, but there is some debate and probably changes coming to ValueRecorder. It's probably best to wait until https://github.com/open-telemetry/opentelemetry-specification/issues/636 is resolved.

**Comment by amitripshtos** [comment]
Is there any way to still support any kind of valueRecorder at the moment, or only Counter is available?

**Comment by aabmass** [comment]
@amitripshtos right now unfortunately ValueRecorder is not supported and the issue linked above is still under a lot of debate.

**Comment by aabmass** [comment]
Value recorder is now called "Histogram". This should get fixed when we upgrade the exporter to the new stable SDK.
